package.json

```diff
@@ -41,8 +41,9 @@
     "test:coverage:path": "nyc --check-coverage yarn test:unit:path",
     "ganache:start": "./development/run-ganache.sh",
     "sentry:publish": "node ./development/sentry-publish.js",
-    "lint": "prettier --check '**/*.json' && eslint . --ext js,snap --cache && yarn lint:styles",
-    "lint:fix": "prettier --write '**/*.json' && eslint . --ext js --cache --fix",
+    "lint:prettier": "prettier '**/*.json' '!lavamoat/**/policy.json'",
```

Comment:
We already ignore these policies in the `.prettierignore` file. Except now that's broken since policy names were introduced. I'd suggest updating that invalid entry instead of adding it here (the `*` needs to be changed to `**`).

Reply:
Fixed in 0ff072978